css/kizgetdown.css

```diff
@@ -0,0 +1,66 @@
+/* Header */
+
+.header-kizgetdown {
+    background-image: url("../images/kizgetdown-bg.jpg");
+    height: 35rem;
+}
+
+.overlay-kizgetdown {
+    height: 35rem;
+    background-color: rgb(255, 0, 0, 0.3);
+}
+
+.kizgetdown-title {
+    font-size: 5rem;
+    position: absolute;
+    top: 10rem;
+    text-align: center;
+    -webkit-box-sizing: content-box;
+    -moz-box-sizing: content-box;
+    box-sizing: content-box;
+    padding: 20px;
```

Comment:
Here's px again. Avoid them, use `rem`.

Reply:
oki. Some things I have done thru online generators and they put px automaticlly